packages/account/src/Sections/Profile/PhoneVerification/verification-link-expired-modal.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Modal, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { useHistory } from 'react-router';
+import { observer, useStore } from '@deriv/stores';
+import { LabelPairedCircleXmarkLgRegularIcon } from '@deriv/quill-icons';
+
+type TVerificationLinkExpiredModal = {
+    should_show_verification_link_expired_modal: boolean;
+    setShouldShowVerificationLinkExpiredModal: (value: boolean) => void;
+};
+
+const VerificationLinkExpiredModal = observer(
+    ({
+        should_show_verification_link_expired_modal,
+        setShouldShowVerificationLinkExpiredModal,
+    }: TVerificationLinkExpiredModal) => {
+        const history = useHistory();
+        const handleCancelButton = () => {
+            setShouldShowVerificationLinkExpiredModal(false);
+            history.goBack();
+        };
+        const { ui } = useStore();
+        const { is_mobile } = ui;
+
+        return (
+            <Modal
+                isMobile={is_mobile}
+                showHandleBar
+                isOpened={should_show_verification_link_expired_modal}
+                primaryButtonCallback={() => setShouldShowVerificationLinkExpiredModal(false)}
```

Comment:
Should we also create a separate function for this callback as we will add some api call for sending new link? 🙏

Reply:
yeap but the API is not ready for now, will implement that when the API is ready. For now we will temporary use useState to mock closing the modal